pandas/_libs/tslibs/conversion.pyx

```diff
@@ -572,6 +606,40 @@ cdef inline void localize_tso(_TSObject obj, tzinfo tz):
     obj.tzinfo = tz
 
 
+cdef inline bint _infer_tsobject_fold(_TSObject obj, object trans,
+                                      object deltas, int32_t pos):
+    """
+    Infer _TSObject fold property from value by assuming 0 and then setting
```

Comment:
can you add a link to the relevant documentation in std library about fold (here and where appropriate)

Reply:
Also linked PEP 495. Standard library pretty much has one paragraph where they explain what fold is and we have the same stuff content-wise, only rephrased.